lib/context-base.js

```diff
@@ -29,3 +39,7 @@ ContextBase.prototype.getScope = function() {
     method.ctor ||
     method.sharedMethod && method.sharedMethod.ctor;
 };
+
+ContextBase.prototype.setReturnArgByName = function(name, value) {
```

Comment:
When you assign a function to a class/prototype property, the stack trace use the property name as a function name.

```
$ node
> function ContextBase(){}
undefined
> ContextBase.prototype.foo = function(){ throw new Error(); }
[Function]
> new ContextBase().foo()
Error
    at ContextBase.foo (repl:1:47)
    at repl:1:19
    at sigintHandlersWrap (vm.js:22:35)
    at sigintHandlersWrap (vm.js:96:12)
    at ContextifyScript.Script.runInThisContext (vm.js:21:12)
    at REPLServer.defaultEval (repl.js:313:29)
    at bound (domain.js:280:14)
    at REPLServer.runBound [as eval] (domain.js:293:12)
    at REPLServer.<anonymous> (repl.js:504:10)
    at emitOne (events.js:101:20)
```